IPIP/0000-gateway-tar-response-format.md

```diff
@@ -54,7 +54,8 @@ Manually created UnixFS DAGs can be turned into malicious TAR files. For example
 if a UnixFS directory contains a file that points at a relative path outside of
 its root, the unpacking of the TAR file may overwrite local files. In order to prevent
 this, if the UnixFS directory contains a file that points at a relative path outside
-of the root, the TAR file creation **must** fail.
+of the root, the TAR file creation **must** fail using the [Trailer Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Trailer)
+`X-Stream-Error` to indicate the error in a human-readable format.
```

Comment:
Gateway specs should not include things that are not supported by web browsers.  
(JS running on a page or inside  a ServiceWorker should be able to use Gateway)

Due to this, mentions of Trailers have to be removed because of https://github.com/mdn/browser-compat-data/issues/14703

Perhaps we could replace it with  a note that connection should be force-closed – are you able to confirm it generates desired effect (error in network tab) in browsers?